pom.xml

```diff
@@ -32,15 +32,15 @@
 
   <properties>
     <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
-    <druid.version>0.13.0-incubating</druid.version>
+    <druid.version>0.23.0</druid.version>
     <jmh.version>1.9.2</jmh.version>
   </properties>
 
   <dependencies>
     <dependency>
       <groupId>org.apache.druid</groupId>
       <artifactId>druid-api</artifactId>
-      <version>${druid.version}</version>
+      <version>0.13.0-incubating</version>
```

Comment:
do we need this dependency given that `druid-api` doesn't exist now?

Reply:
You're right @rohangarg, I've removed this dependency and added NullHandling for `ExampleByteBufferInputRowParserTest` and it's working fine.